config/lava/chromeos/cros-tast.jinja2

```diff
@@ -4,7 +4,7 @@
 - test:
     namespace: chromeos
     timeout:
-      minutes: 5
+      minutes: 15
```

Comment:
The job took 11min to run, I think the timeout should be set to something like 30min. In fact this should be a template variable like for other test plans.

Reply:
Done, updated

Reply:
Thanks. Then maybe as a follow-up this could be turned into a template parameter with a new PR.